### DISCUSSION TITLE
Welcome to Brython Discussions!

### DISCUSSION THREAD

**Discussion by PierreQuentel** [brython-dev/brython#2138] [Announcements]
Let's try to discuss Brython topics in this Discussions area.

**Comment by NadieFiind** [comment]
Hello! Is Brython ever gonna have a build for production? We've created a framework on top of Brython, but it can't be used on production-level applications due to Brython's limitations.

**Comment by PierreQuentel** [comment]
Hello,

I don't know what you mean by "a build for production", but could you open a new discussion instead of commenting the Welcome message ?
